models/device.go

```diff
@@ -31,7 +31,7 @@ type Device struct {
 	Labels         []string                      `json:"labels"`         // Other labels applied to the device to help with searching
 	Location       interface{}                   `json:"location"`       // Device service specific location (interface{} is an empty interface so it can be anything)
 	Service        DeviceService                 `json:"service"`        // Associated Device Service - One per device
-	Profile        DeviceProfile                 `json:"profile"`        // Associated Device Profile - Describes the device
+	ProfileName    string                        `json:"profileName""`	 // Associated Device Profile Name
```

Comment:
Syntax error on this line with trailing double quotes

Reply:
Hi @tsconn23, thanks for your review. Unfortunately, most probably for today Core meeting I will be late. Hope, I will join for the second half.